app/src/components/account/UserMenu.vue

```diff
@@ -0,0 +1,307 @@
+        </div>
+        <div
+          v-for="address in addresses"
+          :key="address.address"
+          class="menu-list-item address-list"
+          :class="{ selected: address.address === selectedAddress }"
+          @click="selectAddress(address.address)"
+        >
+          <div>
+            <span v-if="address.type">{{ address.type }}</span>
+            <span class="address">{{ address.address | formatAddress }}</span>
+            <span class="address-network">{{ address.network }}</span>
+          </div>
+
+          <i v-if="address.address === selectedAddress" class="material-icons"
+            >check</i
+          >
+        </div>
+        <div
+          v-if="session.experimentalMode"
+          class="menu-list-item"
+          :class="{ dark: selectedOption === `create` }"
+          @click="selectOption(`create`)"
+        >
+          <span>Create New Account</span>
+          <i class="material-icons">add_circle</i>
+        </div>
+        <div
+          v-if="session.experimentalMode"
+          class="menu-list-item"
+          :class="{ dark: selectedOption === `settings` }"
+          @click="selectOption(`settings`)"
+        >
+          <span>Settings</span>
+          <i class="material-icons">settings</i>
+        </div>
+        <div
+          v-if="session.experimentalMode"
+          class="menu-list-item"
+          :class="{ dark: selectedOption === `payment` }"
+          @click="selectOption(`payment`)"
+        >
+          <span>Manage Subscription</span>
+          <i class="material-icons">payment</i>
+        </div>
+        <div
+          class="menu-list-item outline"
+          :class="{ dark: selectedOption === `logout` }"
+          @click="
+            selectOption(`logout`)
+            signOut()
+            closePopover()
+          "
+        >
+          <span>Logout</span>
+          <i class="material-icons">exit_to_app</i>
+        </div>
+      </template>
+    </v-popover>
+  </div>
+</template>
+
+<script>
+import Avatar from "common/Avatar"
+import TmBtn from "common/TmBtn"
+import UserMenuAddress from "account/UserMenuAddress"
+import { formatAddress } from "src/filters"
+import { mapGetters, mapState } from "vuex"
+export default {
+  name: `user-menu`,
+  filters: {
+    formatAddress,
+  },
+  components: {
+    Avatar,
+    TmBtn,
+    UserMenuAddress,
+  },
+  data: () => ({
+    addresses: [
+      {
+        address: "cosmos1...abcd",
+        network: "Cosmos Hub",
+      },
+      {
+        address: "akash1...efgh",
+        network: "Akash Testnet",
+      },
+      {
+        address: "polka1...ijkl",
+        network: "Polkadot",
+      },
+    ],
+    selectedAddress: "cosmos1...abcd",
+    selectedOption: "",
+  }),
+  computed: {
+    ...mapState([`session`, `account`]),
+    ...mapGetters([`address`, `currentNetwork`]),
+    user() {
+      return this.account.userSignedIn ? this.account.user : null
+    },
+  },
+  watch: {
```

Comment:
hey @Bitcoinera why do we need a `watch` here?

Reply:
every time a new address is registered on connection it gets added to the UserMenu. Right now it is very primitive though and it serves only as a mean of displaying them (and get the cool pics like the last one I posted 😛 )

But the idea is to, in another PR, you would be able to jump from one to another just by clicking it.

I could also add that one here